docs/guide/Titanium_SDK/Titanium_SDK_Guide/Titanium_Command-Line_Interface_Reference/Titanium_CLI_Plugins.md

```diff
@@ -462,7 +462,7 @@ Indicates if tab completion is enabled or not.
 Information reported by the `titanium config` command as a JSON object.
 
 **env**: Object
-Information about the Titanium build environment similar to `titanium info -t titanium`.
+Information about the Titanium build environment similar to `titanium info -t app`.
```

Comment:
What is `titanium info -t app`? The only valid types are `'all`, `os`, '`odejs`, `titanium`, `osx`, `jdk`, and `haxm`.